README.md

```diff
@@ -3,19 +3,19 @@
 [![Travis Build Status](https://travis-ci.org/junaruga/rpm-py-installer.svg?branch=master)](https://travis-ci.org/junaruga/rpm-py-installer)
 [![Shippable Build Status](https://api.shippable.com/projects/5bb13c0997da11060049f4ad/badge?branch=master)](https://app.shippable.com/github/junaruga/rpm-py-installer/runs?branchName=master)
 
-`rpm-py-installer` is to enable [the RPM Python binding](https://github.com/rpm-software-management/rpm/tree/master/python) in any Python environment.
+`rpm-py-installer` is used to enable [the RPM Python bindings](https://github.com/rpm-software-management/rpm/tree/master/python) in any Python environment.
```

Comment:
I want to remove the word "used". Because the headline's copy is inspired from below my favorite company's vision. "to enable" is used. As the company is UK base, not using "used" is still natural.

https://nanoporetech.com/
> Our goal is to enable the analysis of any living thing, by anyone, anywhere.

You see "An installer to enable the RPM Python binding in any environment." is used as this repository's head line.
This is my favorite head line copy.

"bindings" is better than "binding"? "binding" is used as below wikipadia page's title.
https://en.wikipedia.org/wiki/Language_binding

Reply:
> I want to remove the word "used". Because the headline's copy is inspired from below my favorite company's vision. "something is to enable" is used. As the company is UK base, I think not using the word "used" is still natural expression.
> 
> https://nanoporetech.com/
> 
> > About us
> > Our goal is to enable the analysis of any living thing, by anyone, anywhere.
> 
> You see "An installer to enable the RPM Python binding in any environment." is used as this repository's head line.

Sure, I'll change it back.

> "bindings" is better than "binding"? "binding" is used as below Wikipedia page's title.
> https://en.wikipedia.org/wiki/Language_binding

Indeed it is, I was not aware of the possible usage in singular form. I've mostly heard it used in plural "bindings", however google is not entirely conclusive whether it should be binding or bindings (I have the feeling that bindings is a little more popular, but that might be wishful thinking on my part).

Reply:
Sure, but for "bindings", I want to change "bindings" to "binding" back in this PR. Because as I wrote following message, there are additional parts to replace "binding" with "bindings" in both md files and py files. It's a lot. I think the replacement is not this PR.

Reply:
I've changed these back.